src/cli/format.js

```diff
@@ -241,6 +245,31 @@ async function createIgnorerFromContextOrDie(context) {
   }
 }
 
+/**
+ * This function starts reading any input which user types on the console.
+ * After the user clicks ctrl+D to exit, the input is formatted and printed
+ * on the screen.
+ * @returns {Promise<string>} On success, resolves the input code joined together 
+ * to form a single string. On error, reject with an error message
+ */
+function getInputFromTerminal() {
+  return new Promise((resolve, reject) => {
+    printToScreen("Start writing your code snippet below");
+    printToScreen("After finishing press ctrl+D to exit the read mode");
+    const inputCode = [];
+    const readLine = readline.createInterface({ input, output });
```

Comment:
Are you sure this is correct? Are we piping `stdin` to `stdout`?

Reply:
removed the `output` from the interface.